core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -566,32 +593,36 @@ private void applyRename(RenameEntry entry) {
     mInodeStore.writeInode(inode);
 
     if (oldParent == newParent) {
-      updateLastModifiedAndChildCount(oldParent, entry.getOpTimeMs(), 0);
+      updateTimestampsAndChildCount(oldParent, entry.getOpTimeMs(), 0);
     } else {
-      updateLastModifiedAndChildCount(oldParent, entry.getOpTimeMs(), -1);
-      updateLastModifiedAndChildCount(newParent, entry.getOpTimeMs(), 1);
+      updateTimestampsAndChildCount(oldParent, entry.getOpTimeMs(), -1);
+      updateTimestampsAndChildCount(newParent, entry.getOpTimeMs(), 1);
     }
   }
 
   /**
-   * Updates the last modified time (LMT) for the indicated inode directory, and updates its child
+   * Updates the timestamps for the indicated inode directory, and updates its child
```

Comment:
can you add what the "timestamps" are? (last modified, and last accessed)

Reply:
Added.